index.js

```diff
@@ -79,8 +125,20 @@ genderCount(data);
 // ***************************************************************************
 
 const promo20 = users => {
-  // Your code goes here ...
+  const money = [];
+  for(let user of users){
+    let str = user.balance
+    let num = parseInt(str.replace(/[^\d\.\-]/g, ""))
```

Comment:
not a tip on the regex, because its not something i know a lot, so I would have to google a lot just to figure out what this does. Whenever you do this, always leave a comment.
People are not fluent in regex, so make their (or yours in a week) job easier.
Leave a comment to yourself 💪